### PR TITLE
Update russian.lua

### PR DESCRIPTION
**Pull request by egn1us**
Translated missing strings at the end and fixed some semantic errors.

### CODE REVIEW

**Comment by TimGoll** [comment]
This is really bad timing. I completely overhauled the f1 settings and changed many language strings. In a few days it will be probably merged into the master.

We could merge this for now if alf plans on releasing a 6.4 update.

I'd really appreciate it if you'd translate the missing stuff in a few weeks once the UI update is done

**Comment by egn1us** [comment]
Yes, i can translate it

File: gamemodes/terrortown/gamemode/shared/lang/russian.lua [hunk 1]
```diff
@@ -1213,3 +1213,29 @@ L.disable_spectatorsoutline_tip = "Выключает контуры объек
...
+L.door_locked = "Эта дверь заперта"
+
+-- 2020-02-11
+L.automoved_to_spec = "(АВТОМАТИЧЕСКОЕ СООБЩЕНИЕ) Я был перемещен в наблюдатели потому что был AFK."
+
+-- 2020-02-16
+L.door_auto_closes = "Эта дверь закрывается автоматически."
+L.door_open_touch = "Войти в дверь чтобы она открылась."
+L.door_open_touch_and_use = "Войти в дверь или нажать [{usekey}] чтобы ее открыть."
+L.hud_health = "ХП"
```
Comment (Satton2) [inline gamemodes/terrortown/gamemode/shared/lang/russian.lua:1241]: «Health» was cut to «HP» and transliterated into «ХП». This cannot be called a translation at all.

File: gamemodes/terrortown/gamemode/shared/lang/russian.lua [hunk 2]
```diff
@@ -1213,3 +1213,29 @@ L.disable_spectatorsoutline_tip = "Выключает контуры объек
...
+L.door_close = "Нажмите [{usekey}] чтобы закрыть дверь."
+L.door_locked = "Эта дверь заперта"
+
+-- 2020-02-11
+L.automoved_to_spec = "(АВТОМАТИЧЕСКОЕ СООБЩЕНИЕ) Я был перемещен в наблюдатели потому что был AFK."
+
+-- 2020-02-16
+L.door_auto_closes = "Эта дверь закрывается автоматически."
+L.door_open_touch = "Войти в дверь чтобы она открылась."
+L.door_open_touch_and_use = "Войти в дверь или нажать [{usekey}] чтобы ее открыть."
```
Comment (Satton2) [inline gamemodes/terrortown/gamemode/shared/lang/russian.lua:1240]: The same issues as in the string above.

File: gamemodes/terrortown/gamemode/shared/lang/russian.lua [hunk 3]
```diff
@@ -1213,3 +1213,29 @@ L.disable_spectatorsoutline_tip = "Выключает контуры объек
...
+L.door_open = "Нажмите [{usekey}] чтобы открыть дверь."
+L.door_close = "Нажмите [{usekey}] чтобы закрыть дверь."
+L.door_locked = "Эта дверь заперта"
+
+-- 2020-02-11
+L.automoved_to_spec = "(АВТОМАТИЧЕСКОЕ СООБЩЕНИЕ) Я был перемещен в наблюдатели потому что был AFK."
+
+-- 2020-02-16
+L.door_auto_closes = "Эта дверь закрывается автоматически."
+L.door_open_touch = "Войти в дверь чтобы она открылась."
```
Comment (Satton2) [inline gamemodes/terrortown/gamemode/shared/lang/russian.lua:1239]: «Enter the door to open it», excuse me what?
Also, the same issue with the word «чтобы» as above.

File: gamemodes/terrortown/gamemode/shared/lang/russian.lua [hunk 4]
```diff
@@ -1213,3 +1213,29 @@ L.disable_spectatorsoutline_tip = "Выключает контуры объек
...
+L.hold_aim_tip = "Вы будете продолжать прицеливаться пока вы удерживаете кнопку второстепенной атаки (по умолчанию: ПКМ)"
+
+-- 2020-02-09
+L.name_door = "Дверь"
+L.door_open = "Нажмите [{usekey}] чтобы открыть дверь."
+L.door_close = "Нажмите [{usekey}] чтобы закрыть дверь."
+L.door_locked = "Эта дверь заперта"
+
+-- 2020-02-11
+L.automoved_to_spec = "(АВТОМАТИЧЕСКОЕ СООБЩЕНИЕ) Я был перемещен в наблюдатели потому что был AFK."
```
Comment (Satton2) [inline gamemodes/terrortown/gamemode/shared/lang/russian.lua:1235]: «потому что» requires a comma before it.
Also, usually, the abbreviation «AFK» translates as «бездействие».

File: gamemodes/terrortown/gamemode/shared/lang/russian.lua [hunk 5]
```diff
@@ -1213,3 +1213,29 @@ L.disable_spectatorsoutline_tip = "Выключает контуры объек
...
+L.disable_doubletap_sprint_tip = "Двойное нажатие на клавишу движения больше не будет вас заставлять бегать"
+
+-- 2020-02-03
+L.hold_aim = "Удерживать чтобы прицелиться"
+L.hold_aim_tip = "Вы будете продолжать прицеливаться пока вы удерживаете кнопку второстепенной атаки (по умолчанию: ПКМ)"
+
+-- 2020-02-09
+L.name_door = "Дверь"
+L.door_open = "Нажмите [{usekey}] чтобы открыть дверь."
+L.door_close = "Нажмите [{usekey}] чтобы закрыть дверь."
```
Comment (Satton2) [inline gamemodes/terrortown/gamemode/shared/lang/russian.lua:1231]: The same issue with the word «чтобы» as above.

File: gamemodes/terrortown/gamemode/shared/lang/russian.lua [hunk 6]
```diff
@@ -1213,3 +1213,29 @@ L.disable_spectatorsoutline_tip = "Выключает контуры объек
...
+-- 2020-01-04
+L.doubletap_sprint_anykey = "Продолжать бегать двойным нажатием пока вы не остановитесь"
+L.doubletap_sprint_anykey_tip = "Вы будете продолжать бегать пока вы движитесь"
+
+L.disable_doubletap_sprint = "Отключить бег двойным нажатием"
+L.disable_doubletap_sprint_tip = "Двойное нажатие на клавишу движения больше не будет вас заставлять бегать"
+
+-- 2020-02-03
+L.hold_aim = "Удерживать чтобы прицелиться"
+L.hold_aim_tip = "Вы будете продолжать прицеливаться пока вы удерживаете кнопку второстепенной атаки (по умолчанию: ПКМ)"
```
Comment (Satton2) [inline gamemodes/terrortown/gamemode/shared/lang/russian.lua:1226]: The same issue with the word «пока» as above. Word «чтобы» also requires a comma.

File: gamemodes/terrortown/gamemode/shared/lang/russian.lua [hunk 7]
```diff
@@ -1213,3 +1213,29 @@ L.disable_spectatorsoutline_tip = "Выключает контуры объек
 
 L.disable_overheadicons = "Выключить иконки ролей."
 L.disable_overheadicons_tip = "Выключает иконки ролей над головами игроков (прибавка к производительности)."
+
+-- 2020-01-04
+L.doubletap_sprint_anykey = "Продолжать бегать двойным нажатием пока вы не остановитесь"
+L.doubletap_sprint_anykey_tip = "Вы будете продолжать бегать пока вы движитесь"
```
Comment (Satton2) [inline gamemodes/terrortown/gamemode/shared/lang/russian.lua:1219]: The word «пока» is a conjunction in these strings which requires a comma before it.

File: gamemodes/terrortown/gamemode/shared/lang/russian.lua [hunk 8]
```diff
@@ -231,7 +231,7 @@ L.search_eyes = "Используя свои детективные навыки
 
 
 -- Scoreboard
-L.sb_playing = "Вы играете на..."
+L.sb_playing = "Вы играете на"
```
Comment (Satton2) [inline gamemodes/terrortown/gamemode/shared/lang/russian.lua:234]: Why the ellipsis was removed?

**Comment by TimGoll** [comment]
I reject this since Satton2 said these translations are not good and they aren't needed in the next update anymore.

**Comment by Histalek** [comment]
As Satton has pointed out this translation has serious Problems as well as the strings will most likely be deprecated in the next update anyway. Therefore i'll close this PR.

**Pull request closed without merging**